base/src/expressions/parser/test_stringify.rs

```diff
@@ -0,0 +1,146 @@
+use super::Parser;
```

Comment:
Hi @jaycarlton , these tests are amazing? But why did you add them? Did you add them because you want to learn how to use the API or because you are covering some case that wasn't covered before. If it is the second case we definitely need them in!